### PR TITLE
feat: calibration align language

### PR DESCRIPTION
**Pull request by andrushechka-b**
This pr implements language alignment for calibration tool.

### CODE REVIEW

**Review by lvjonok — APPROVED** [review]
Looks good

File: src/calibration/model.py [hunk 1]
```diff
@@ -2,5 +2,30 @@
...
+            btnFinish='Finish',
+        ),
+        ru=dict(
+            window='Калибровка',
+            btnNext='Далее',
+            btnCancel='Отмена',
+            btnFinish='Завершить',
+        ),
+    )
+
```
Comment (l1va) [inline src/calibration/model.py:19]: why do you want to use a separate place, not the common. Not the common approach?

Comment (andrushechka-b) [reply src/calibration/model.py:19]: I intended to use separate place in order to keep the calibration staff as a separate module. Now it is very easy to make calibration to be a standalone program. Probably it is time to start mix it with a main program, do you confirm?

Comment (l1va) [reply src/calibration/model.py:19]: not really, it still can be a separate module, in a separate folder with separate translations, but with the common parent class.

Comment (andrushechka-b) [reply src/calibration/model.py:19]: Sorry, what common parent class do you mean? My idea was to import _src.locales_ module and use its _getLocale()_  function

Comment (l1va) [reply src/calibration/model.py:19]: yes, let's create locale pkg. Here I meant that you can create a class with translations, not just dicts.

**Review by l1va — APPROVED** [review]

**Pull request closed without merging**